go/vt/sqlparser/ast.go

```diff
@@ -1802,9 +1802,9 @@ type (
 
 	// BetweenExpr represents a BETWEEN or a NOT BETWEEN expression.
 	BetweenExpr struct {
-		Operator RangeCondOperator
-		Left     Expr
-		From, To Expr
+		isBetween bool
```

Comment:
this should be a public field, so we can inspect it from outside the parser package